lookup_tables/ipinfo/ipinfo_asn.yml

```diff
@@ -90,8 +90,6 @@ LogTypeMap:
         - 'UserIp'
     - LogType: Crowdstrike.DetectionSummary
       Selectors:
-        - 'LocalIP'
```

Comment:
This is obviously wrong. Need to revert.

Reply:
👍